extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss

```diff
@@ -91,5 +92,142 @@
 	display: none;
 }
 
+.global-navigation-2016 {
+	.global-nav-search-input-wrapper .autocomplete-container,
+	.notifications-container .notifications-container,
+	.account-navigation-container .user-menu,
+	.AccountNavigation .UserLoginDropdown {
+		margin-top: 1px;
+	}
+
+	.wikia-logo {
+		display: flex;
+		align-items: flex-end;
+		text-decoration: none !important;
```

Comment:
I believe we should leave it. We don't want to get the "HOME OF FANDOM" link underlined by some user/wiki/old styles. 

`!important` only shows how badly we don't want it to happen